src/main/java/edu/kis/powp/command/SetPositionCommand.java

```diff
@@ -0,0 +1,19 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class SetPositionCommand implements DriverCommand{
+    public int x, y;
+    public final Job2dDriver driver;
+
+    public SetPositionCommand(int x, int y, Job2dDriver driver) {
```

Comment:
The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).